lib/pbio/include/pbio/protocol.h

```diff
@@ -277,6 +277,12 @@ typedef enum {
      * @since Pybricks Profile v1.2.0
      */
     PBIO_PYBRICKS_STATUS_SHUTDOWN_REQUEST = 8,
+    /**
+     * Hub is connected to a host (like Pybricks Code) via BLE.
```

Comment:
Do we also need a peripheral connected status?

Reply:
Deferred to https://github.com/pybricks/support/issues/1716